skglm/solvers/base.py

```diff
@@ -0,0 +1,41 @@
+from abc import abstractmethod
+
+
+class BaseSolver():
```

Comment:
shouldn't we add also a ``path`` method?

Reply:
No we decided not to include it as it does not make sense for non-convex estimators (the path is not unique).